### PR TITLE
Keyword view

### LINKED ISSUE #64: Showing keyword in recommendation list
Create a custom view with a blue hashtag before the keyword name and surrounded with a white oval background to show the keyword. Take in consideration the keywords will be clickable

### PR DESCRIPTION
**Pull request by icerrate**
Close #64

### CODE REVIEW

File: app/src/main/java/com/belatrixsf/allstars/entities/Keyword.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by gyosida on 4/12/16.
+ */
+public class Keyword {
```
Comment (PedroCarrillo) [inline app/src/main/java/com/belatrixsf/allstars/entities/Keyword.java:26]: @gyosida is that yout?

Comment (gyosida) [inline app/src/main/java/com/belatrixsf/allstars/entities/Keyword.java:26]: I guess he is trying to be like me ? xD

File: app/src/main/java/com/belatrixsf/allstars/ui/common/views/KeywordView.java [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        initView();
+    }
+
+    public KeywordView(Context context, AttributeSet attrs) {
+        super(context, attrs);
+        initView();
+        TypedArray typedArray = context.obtainStyledAttributes(attrs, R.styleable.KeywordView);
+
+        String hintText = typedArray.getString(R.styleable.KeywordView_key_text);
+        keyboardTextView.setText(hintText);
```
Comment (PedroCarrillo) [inline app/src/main/java/com/belatrixsf/allstars/ui/common/views/KeywordView.java]: hintText? :D

File: app/src/main/java/com/belatrixsf/allstars/ui/common/views/KeywordView.java [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+        typedArray.recycle();
+    }
+
+    private void initView() {
+        LayoutInflater inflater = (LayoutInflater) getContext().getSystemService(Context.LAYOUT_INFLATER_SERVICE);
+        inflater.inflate(R.layout.view_keyword, this, true);
+        keyboardTextView = (TextView) findViewById(R.id.keyword);
+    }
+
+    public void setData(String data) {
```
Comment (gyosida) [inline app/src/main/java/com/belatrixsf/allstars/ui/common/views/KeywordView.java]: we can use a more descriptive method name. eg: setKeyword(String keyword)

File: app/src/main/java/com/belatrixsf/allstars/ui/common/views/KeywordView.java [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+        if (data != null && !data.isEmpty()) {
+            keyboardTextView.setText(data);
+            value = data;
```
Comment (gyosida) [inline app/src/main/java/com/belatrixsf/allstars/ui/common/views/KeywordView.java]: do we really need to have a global reference for this string? or later when saving we could get the text from the TextView.gettext()?

File: app/src/main/res/drawable/background_keyword_press.xml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (gyosida) [inline app/src/main/res/drawable/background_keyword_press.xml]: background_keyword_pressed

**Pull request merged**